.github/workflows/build-and-test.yml

```diff
@@ -11,6 +11,11 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
 
+      - name: Set up Node 16
```

Comment:
Looks fine to me. Or should we use a different build image @aaron-steinfeld?

Reply:
ubuntu 20.04 is pretty much their newest, I believe https://github.com/actions/virtual-environments#available-environments - and it uses `Npm 6.14.14`, `Node 14.17.5`

Reply:
(I'm assuming updating node in this way updates npm too)

Reply:
Yep. it should install npm

Reply:
going to merge this.